examples/jupiter-swap/src/api/jupiter.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  type PublicKey,
+  VersionedTransaction,
+  type Cluster,
+  clusterApiUrl,
+  Connection,
+} from '@solana/web3.js';
+import type {
+  JupiterQuoteResponse,
+  JupiterSwapResponse,
+  JupiterToken,
+  Token,
+  TokenType,
+} from '../types';
+
+const getApiUrl = () => {
+  const url = process.env.JUPITER_API_URL;
+  if (!url) {
+    throw new Error('JUPITER_API_URL is not set');
+  }
+  return url;
+};
+
+const getTokenListEndpoint = () => {
+  return (
+    process.env.JUPITER_TOKEN_LIST_ENDPOINT || 'https://token.jup.ag/strict'
```

Comment:
if you create a config file like we did in the FE repo, you can actually have this condition there. it's just a thought, feel free to ignore. https://github.com/dscvr-one/dscvr-fe/blob/develop/src/shared/lib/config/index.ts

Reply:
WDYT @ncpenke, should we go with a config file? What would be the simplest for an almost non developer user?

Reply:
I think our current audience should be someone who's proficient in TypeScript. Probably a no-code solution might be more appropriate for a non-developer,  so I think following our usual best practices is ok.

Reply:
Talked to Chandra, we will leave it hardcoded for now